gammapy/maps/tests/test_region.py

```diff
@@ -288,4 +289,13 @@ def test_get_wcs_coord(region):
     region_coords = geom.get_wcs_coord()
     coord = SkyCoord("100d", "30d")
     assert geom.contains(region_coords.skycoord[0])
-    assert geom.contains(coord) is not True
\ No newline at end of file
+    assert geom.contains(coord) is not True
```

Comment:
I know it's unrelated but this should rather read `assert not gom.contains(coord)`